mutiny-wasm/src/lib.rs

```diff
@@ -346,6 +346,27 @@ impl MutinyWallet {
             .to_string())
     }
 
+    #[wasm_bindgen]
+    pub async fn send_payjoin(
+        &self,
+        payjoin_uri: String,
+        amount: u64,      /* override the uri amount if desired */
+        labels: &JsValue, /* Vec<String> */
+        fee_rate: Option<f32>,
+    ) -> Result<String, MutinyJsError> {
+        // I know walia parses `pj=` and `pjos=` but payjoin::Uri parses the whole bip21 uri
```

Comment:
I think this is fine, for mutiny-web we can just check if the pj uri is defined, and if it is then we use `send_payjoin` instead of `send_to_address`